index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="EN">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Apple store</title>
+  <link rel="stylesheet" href="./styles/index.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <div class="main-page">
+    <div class="navbar-container">
+      <div class="navbar">
+        <div><img src="./img/apple-img.png" height="25" alt="apple image" class="apple-image" /></div>
+        <div><a href="#">Store</a></div>
+        <div><a href="#">Mac</a></div>
+        <div><a href="#">iPod</a></div>
+        <div><a href="#">iPhone</a></div>
+        <div><a href="#">iPad</a></div>
+        <div><a href="#">iTunes</a></div>
+        <div><a href="#">Support</a></div>
+        <div class="search-container"><i class="fa fa-search icon" style="color:white">
```

Comment:
In-line styles are [bad practice](https://speckyboy.com/good-bad-css-practices). All styles should be in the CSS file.

Reply:
Done